bin/run-command

```diff
@@ -88,18 +88,18 @@ overrides=$(cat << EOF
   ]
 }
 EOF
-)
+)"
```

Comment:
I'm not sure about this one? It might because the exception since there's an `EOF` involved